package.json

```diff
@@ -36,38 +36,31 @@
         }
     },
     "dependencies": {
-        "@11ty/eleventy": "0.12.1",
-        "@11ty/eleventy-img": "0.9.0",
-        "@11ty/eleventy-navigation": "0.3.2",
-        "@11ty/eleventy-plugin-rss": "1.1.1",
-        "@11ty/eleventy-plugin-syntaxhighlight": "3.1.2",
-        "eleventy-plugin-fluid": "0.2.1",
-        "infusion": "3.0.0-dev.20210120T204128Z.6e4be079f.FLUID-6580"
+        "@11ty/eleventy": "2.0.1",
+        "@11ty/eleventy-img": "3.1.0",
+        "@11ty/eleventy-navigation": "0.3.5",
+        "@11ty/eleventy-plugin-rss": "1.2.0",
+        "@11ty/eleventy-plugin-syntaxhighlight": "5.0.0",
+        "eleventy-plugin-fluid": "1.2.0",
+        "infusion": "4.6.0"
     },
     "devDependencies": {
         "@babel/core": "7.15.0",
         "@commitlint/cli": "13.1.0",
         "@commitlint/config-conventional": "13.1.0",
-        "backstopjs": "5.3.7",
+        "backstopjs": "6.2.2",
         "cross-env": "7.0.3",
-        "debug": "4.3.2",
-        "fluid-lint-all": "1.1.4",
+        "fluid-lint-all": "1.2.9",
         "html-minifier": "4.0.0",
-        "husky": "4.3.8",
+        "husky": "8.0.3",
```

Comment:
This requires an update to Husky's configuration, see:

https://typicode.github.io/husky/migrating-from-v4.html